buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -280,19 +314,23 @@ class VagrantTestPlugin implements Plugin<Project> {
         }
 
         Task createLinuxRunnerScript = project.tasks.create('createLinuxRunnerScript', FileContentsTask) {
-            dependsOn copyPackagingTests
+            dependsOn copyPackagingTests, linuxGradleJdk, linuxSystemJdk
             file "${testsDir}/run-tests.sh"
             contents """\
                      if [ "\$#" -eq 0 ]; then
                        test_args=( "${-> project.extensions.esvagrant.testClass}" )
                      else
                        test_args=( "\$@" )
                      fi
-                     "\$SYSTEM_JAVA_HOME"/bin/java -cp "\$PACKAGING_TESTS/*" org.elasticsearch.packaging.VMTestRunner "\${test_args[@]}"
+                     
+                     if [ -z "\$SYSTEM_JAVA_HOME" ]; then
+                       export SYSTEM_JAVA_HOME="${-> convertPath(project, linuxSystemJdk.toString()) }"
+                     fi
+                     "${-> convertPath(project, linuxGradleJdk.toString()) }"/bin/java -cp "\$PACKAGING_TESTS/*" org.elasticsearch.packaging.VMTestRunner "\${test_args[@]}"
```

Comment:
As I understand it we are on the path to ditch this `VMTestRunner` in favor of just delegating to Gradle, yes?

Reply:
Yep, just trying to do this in phases.